manifests/n/nektos/act/0.2.35/nektos.act.locale.en-US.yaml

```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: nektos.act
+PackageVersion: 0.2.35
+PackageLocale: en-US
+Publisher: cplee.org
+PublisherUrl: https://cplee.org/
+PublisherSupportUrl: https://github.com/nektos/act#support
+PrivacyUrl: https://cplee.org/
+Author: Casey Lee
+PackageName: act
+PackageUrl: https://github.com/nektos/act
+License: MIT
+LicenseUrl: https://github.com/nektos/act/blob/master/LICENSE
+Copyright: Copyright (C) 2019-2023 Casey Lee
+ShortDescription: Run your GitHub Actions locally!
+Tags:
+- act
+- github
+ReleaseNotesUrl: https://github.com/nektos/act/releases/tag/v0.2.35
+InstallationNotes: https://github.com/nektos/act#installation
```

Comment:
```suggestion
```

Do note that `InstallationNotes` should contain the actual notes rather than the Url.

Reply:
> Do note that `InstallationNotes` should contain the actual notes rather than the Url.

the notes are a bit big so should i remove this section ?

Reply:
Sure, the section only mentions prerequisites and package managers, which should be already handled by Winget.